django/forms/formsets.py

```diff
@@ -40,6 +40,8 @@ class ManagementForm(Form):
     MIN_NUM_FORMS = IntegerField(required=False, widget=HiddenInput)
     MAX_NUM_FORMS = IntegerField(required=False, widget=HiddenInput)
 
+    template_name = "django/forms/table.html"
```

Comment:
I guess we could have the output change but that seems possible breaking change that we can avoid at this point in time. It's hidden anyway so doesn't impact screen reader users?

Reply:
Unless folks override FORM_RENDERER it will be unchanged no? 🤔

(Other way round: if we don't change it now, when are we going to — abruptly at 5.0? — is that any better?)

Reply:
> ... when are we going to ...

My thought was, do we _need_ to change it?